kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/exceptionhandlers/IgnoreRecordTooLargeHandler.java

```diff
@@ -0,0 +1,27 @@
+package org.hypertrace.core.kafkastreams.framework.exceptionhandlers;
+
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.errors.RecordTooLargeException;
+import org.apache.kafka.streams.errors.ProductionExceptionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class IgnoreRecordTooLargeHandler implements ProductionExceptionHandler {
+  private static final Logger LOGGER = LoggerFactory.getLogger(IgnoreRecordTooLargeHandler.class);
+
+  @Override
+  public ProductionExceptionHandlerResponse handle(ProducerRecord<byte[], byte[]> record, Exception exception) {
+    if (exception instanceof RecordTooLargeException) {
```

Comment:
This means the record is dropped?

Reply:
yes, based on `max.request.size` property.